guides/doc-Administering_Red_Hat_Satellite/master.adoc

```diff
@@ -1,11 +1,13 @@
+include::common/attributes.adoc[]
 :imagesdir: images
 :administering-red-hat-satellite:
 :numbered:
 ifeval::["{build}" != "satellite"]
 :toc:
 :toc-placement: left
+:revnumber: {ProjectVersion}
+:revdate: published {docyear}
```

Comment:
Is a year sufficient? Perhaps it needs to be a date?

Reply:
Date is at the end of each page (PDF version does not have it). I can add month but I would like to avoid full date.

In the github action, I actually `sed` date in the format of YYYY-MM-DD HH:MM to simply YYYY-MM because I don't think readers need to know exact date. It creates a change every time we commit into `gh-pages` and I would like to have sane history.

Alternatively, I can introduce some "nightly" variable which will make this to only appear on released branches. As long as we won't forget to flip it that could be the best solution.

Reply:
That variable might be actually a good idea - we could leverage it with values "nightly, stable, old" and utilize it to render some warnings for old and nightly content.

Reply:
I think it makes sense to have smaller changes, but as a reader it implies it's only published yearly. At least a PDF gives that impression.

In our current manual we also have something like "this isn't the latest version, see $stable" and "this is nightly, see $stable for a supported version". Having a similar thing here does make sense to me. Perhaps even something more complex: note when it's supported. Currently Foreman 2.2 and 2.3 are supported, but 2.1 and earlier can get a big warning that the release is end of life. Especially for an update guide that's great to have. Nobody should install an EOL release unless they know what they're doing.